pkg/operator/operator.go

```diff
@@ -149,6 +151,7 @@ func NewOperator(ctx context.Context, operator *operator.Operator) (context.Cont
 	launchTemplateProvider := launchtemplate.NewDefaultProvider(
 		ctx,
 		cache.New(awscache.DefaultTTL, awscache.DefaultCleanupInterval),
+		cache.New(awscache.DefaultTTL, awscache.DefaultCleanupInterval),
```

Comment:
What's this second cache for, should it be the same as the other cache that you're passing through the capacity reservation provider?